gravitybridge/assetlist.json

```diff
@@ -296,6 +296,47 @@
         }
       ]
     },
+    {
+      "description": "Gravity Bridge sDAI",
+      "denom_units": [
+        {
+          "denom": "gravity0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+          "exponent": 0
+        },
+        {
+          "denom": "gsdai",
+          "exponent": 18
+        }
+      ],
+      "base": "gravity0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+      "name": "Savings Dai",
+      "display": "gsdai",
+      "symbol": "sDAI",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/sdai.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/sdai.svg"
+      },
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0x83F20F44975D03b1b09e64809B757c47f942BEeA"
+          },
+          "provider": "Gravity Bridge"
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
```

Comment:
I've also added an image pointer so we only need the one copy and deleted the duplicate images you'd added to gravitybridge's images folder.